src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -0,0 +1,489 @@
+| iEEGCoordinateSystemDescription     | RECOMMENDED. Freeform text description or link to document describing the iEEG coordinate system system in detail (e.g., “Coordinate system with the origin at anterior commissure (AC), negative y-axis going through the posterior commissure (PC), z-axis going to a mid-hemisperic point which lies superior to the AC-PC line, x-axis going to the right”).                                                                                                                                                                           |
+| iEEGCoordinateProcessingDescription | RECOMMENDED. Has any post-processing (such as projection) been done on the electrode positions (e.g., “surface_projection”, “none”).                                                                                                                                                                                                                                                                                                                                                                                                       |
+| iEEGCoordinateProcessingReference   | RECOMMENDED. A reference to a paper that defines in more detail the method used to localize the electrodes and to post-process the electrode positions. .                                                                                                                                                                                                                                                                                                                                                                                  |
+
+### Recommended 3D coordinate systems
+
+It is preferred that electrodes are localized in a 3D coordinate system (with
+respect to a pre- and/or post-operative anatomical MRI or CT scans or in a
+standard space as specified in BIDS Appendix VIII: preferred names of Coordinate
+systems, such as ACPC).
+
+### Allowed 2D coordinate systems
+
+If electrodes are localized in 2D space (only x and y are specified and z is
+n/a), then the positions in this file must correspond to the locations expressed
+in pixels on the photo/drawing/rendering of the electrodes on the brain. In this
+case, coordinates must be (row,column) pairs, with (0,0) corresponding to the
+upper left pixel and (N,0) corresponding to the lower left pixel.
+
+### Multiple coordinate systems
```

Comment:
This is a subsection of `_coordsystem.json`, but it talks about `_electrodes.tsv`. Does it apply to both? Please clarify.

Reply:
again, something for @robertoostenveld @dorahermes @choldgraf

Reply:
Yes, it applies to both.